web/src/engine/dom-utils/src/createUnselectableElement.ts

```diff
@@ -3,7 +3,5 @@ export default function createUnselectableElement<E extends "p"|"style"|"script"
   const e = document.createElement<E>(nodeName);
 
   e.style.userSelect="none";
-  e.style.webkitUserSelect="none";
```

Comment:
I think we still need this. See https://caniuse.com/?search=user-select; Safari 17.6 still requires the prefix.

It's also a functionality change, which I don't like when we are doing code cleanup work.

Reply:
I can force it to be in place then; I only made the change because TypeScript (the latest version) does not provide typings for it, believing it doesn't exist.  It wouldn't be the first time we've had to override them, though.

----

"Fun" fact:  adjusting for this and using `e.style["webkitUserSelect"]` triggers an _instant_ "implicitly type `any`" error.  The cleanest way I know to counter that is with a `// @ts-ignore`... which doesn't feel very clean.

Reply:
... and I stand corrected - it's merely marked as deprecated by TS.  Typings do exist here; I must have been thinking of one of the numerous other cases where I did have to insert that little incantation.